examples/config-gc.json

```diff
@@ -3,10 +3,48 @@
     "storage": {
         "rootDirectory": "/tmp/zot",
         "gc": true,
-        "gcReferrers": true,
         "gcDelay": "2h",
         "untaggedImageRetentionDelay": "4h",
-        "gcInterval": "1h"
+        "gcInterval": "1h",
+        "retention": {
+            "dryRun": false,
```

Comment:
Is dryRun going to just produce a log that this image retention period is over and can be deleted?

Reply:
That was what we had discussed, yes.